samples/apps/copilot-chat-app/README.md

```diff
@@ -69,6 +69,13 @@ and these components are functional:
               option is appropriate for your instance.    
             * You will insert your Azure endpoint key during build of the backend
               API Server
+         
+         * If you are using speech to text as input option then under the `AzureSpeechConfig` block, make sure the following configuration
```

Comment:
If you are using speech-to-text as an input option, then under the `AzureSpeechConfig` block, make sure the following configuration matches your instance